django/core/signing.py

```diff
@@ -269,12 +277,28 @@ def unsign(self, value, max_age=None):
         """
         result = super().unsign(value)
         value, timestamp = result.rsplit(self.sep, 1)
-        timestamp = b62_decode(timestamp)
-        if max_age is not None:
-            if isinstance(max_age, datetime.timedelta):
-                max_age = max_age.total_seconds()
-            # Check timestamp is not older than max_age
-            age = time.time() - timestamp
-            if age > max_age:
-                raise SignatureExpired("Signature age %s > %s seconds" % (age, max_age))
+        self.payload_timestamp = b62_decode(timestamp)
+        self._verify_max_age(max_age)
+        return value
+
+    def unsign_object(self, signed_obj, max_age=None, **kwargs):
+        if self.expiration_key is None:
+            return super(TimestampSigner, self).unsign_object(
+                signed_obj, max_age=max_age, **kwargs
+            )
```

Comment:
Can we use `super()`? I don't see any test failures when changing to the:
```suggestion
            return super().unsign_object(signed_obj, max_age=max_age, **kwargs)
```